modules/vye/app/policies/vye/user_info_policy.rb

```diff
@@ -9,5 +9,29 @@ def create?
     end
 
     alias_method :show?, :create?
+
+    def claimant_lookup?
```

Comment:
Can these policies be combined since they are the same?

Reply:
In the controller you can call one policy:

modules/vye/app/controllers/vye/v1/dgib_verifications_controller.rb
`before_action { authorize :vye, :access? }`

app/policies/vye_policy.rb ### NEW POLICY FILE
```
  def access?
      return true if user.present?

      false
  end
```